tests/notificationprofile/test_views.py

```diff
@@ -3,67 +3,85 @@
+        source_type = SourceSystemTypeFactory(name="nav")
+        source1_user = SourceUserFactory(username="nav1")
+        self.source1 = SourceSystemFactory(name="NAV 1", type=source_type, user=source1_user)
+
+        timeslot1 = TimeslotFactory(user=self.user1, name="Never")
         self.timeslot2 = TimeslotFactory(user=self.user1, name="Never 2: Ever-expanding Void")
         filter1 = FilterFactory(
             user=self.user1,
             name="Critical incidents",
             filter_string=f'{{"sourceSystemIds": [{self.source1.pk}]}}',
         )
-        self.notification_profile1 = NotificationProfileFactory(user=self.user1, timeslot=self.timeslot1)
+        self.notification_profile1 = NotificationProfileFactory(user=self.user1, timeslot=timeslot1)
         self.notification_profile1.filters.add(filter1)
+
         # Default email destination is automatically created with user
         self.synced_email_destination = self.user1.destinations.get()
-        self.non_synced_email_destination = DestinationConfigFactory(
-            user=self.user1,
-            media=Media.objects.get(slug="email"),
-            settings={"email_address": "test@example.com", "synced": False},
-        )
-        self.sms_destination = DestinationConfigFactory(
-            user=self.user1,
-            media=Media.objects.get(slug="sms"),
-            settings={"phone_number": "+4747474747"},
-        )
         self.notification_profile1.destinations.set([self.synced_email_destination])
 
     def teardown(self):
         connect_signals()
 
-    def test_can_get_all_incidents_of_notification_profile(self):
-        response = self.user1_rest_client.get(
-            path=f"/api/v2/notificationprofiles/{self.notification_profile1.pk}/incidents/"
-        )
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(len(response.data), 1)
-        self.assertEqual(response.data[0]["pk"], self.incident1.pk)
+    def test_notification_profile_list_is_reachable(self):
+        response = self.user1_rest_client.get(path=self.ENDPOINT)
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+
+    def test_notification_profile_list_has_all_user_profiles(self):
+        response = self.user1_rest_client.get(path=self.ENDPOINT)
+        all_profile_pks = set(profile.pk for profile in self.user1.notification_profiles.all())
+        all_response_pks = set(profile["pk"] for profile in response.data)
+        self.assertEqual(all_response_pks, all_profile_pks)
+
+    def test_specific_notification_profile_is_reachable(self):
+        profile1_pk = self.notification_profile1.pk
+        profile1_path = f"{self.ENDPOINT}{profile1_pk}/"
+        response = self.user1_rest_client.get(path=profile1_path)
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+
+    def test_specific_notification_profile_is_what_we_think_it_is(self):
```

Comment:
This test name made me LOL, but even such as simple test is quite hard to describe (i.e. "when we ask for profile X, the API should return profile X" - it sounds almost laughable)

```suggestion
    def test_get_notification_profile_should_return_the_profile_we_asked_for(self):
```